mobipick_gazebo/launch/mobipick/tables_demo.launch

```diff
@@ -4,6 +4,9 @@
   <arg name="tf_prefix" default="$(arg namespace)" doc="tf_prefix to be used" />
   <arg name="robot_name" default="mobipick" doc="Sets the name of the robot in gazebo" />
 
+  <arg name="world" default="pbr_moelk"/>
```

Comment:
What I currently unintuitive, is that options are provided for the `world config` but not for the `world` argument itself and consequently also the robot pose. This makes the impression that you can change the `world_config` to one of the three values and everything will work.

Reply:
added documentation in https://github.com/DFKI-NI/mobipick/commit/2fbb7af21533511051f4d1600f5cb245566b9305 and https://github.com/DFKI-NI/mobipick_labs/commit/b878830224d4c8450a2c350115ac60c944d25dee , does this solve the issue?